src/blog/another blog piece.md

```diff
@@ -9,14 +9,18 @@ tags:
 
 This is an intro!!!
 
-### Big fuckin heading
+# Big fuckin heading
```

Comment:
Sorry also wasn't clear enough before. The `title` in the metadata at the top is rendered as the page's `h1` in the big blue header (and the title in the browser tab). So you shouldn't have your own extra `h1` in the post content

Reply:
Oh okay so for headings within the article it would be two #s for the biggest heading?

Reply:
Yup. But try not to think of it as a way to control size, since I could easily write CSS to make an `h2` bigger than an `h6`. It's about the hierarchy of the content. `h2` makes a section, `h3` makes a subsection within that, `h4` makes a subsection within that etc. Don't skip levels, since that's confusing for machines/assistive tech to understand. i.e. You shouldn't follow an `h2` with an `h4` just because you prefer the size.

Here's an example of using a screenreader to see a summary of the page structure based on the heading levels. Think of this as how a visual user glances at the headings (or table of contents) to get a sense of the structure of an article.

![voiceover-headings](https://user-images.githubusercontent.com/9408641/107236301-7ab1ca80-6a1d-11eb-8928-7a01b831c170.jpg)

Reply:
Got it. Makes a lot of sense! Thanks